packages/orbit-components/src/Seat/README.md

```diff
@@ -34,11 +34,13 @@ Table below contains all types of the props available in Seat component.
 
 Table below contains all types of the props available in Seat/SeatLegend component.
 
-| Name     | Type                  | Default   | Description                         |
-| :------- | :-------------------- | :-------- | :---------------------------------- |
-| dataTest | `string`              |           | Optional prop for testing purposes. |
-| type     | [`enum`](#modal-enum) | `default` | Visual type of SeatLegend           |
-| label    | `string`              |           | Label text inside of a SeatLegend   |
+| Name       | Type                  | Default   | Description                                                                           |
+| :--------- | :-------------------- | :-------- | :------------------------------------------------------------------------------------ |
+| dataTest   | `string`              |           | Optional prop for testing purposes.                                                   |
+| id         | `string`              |           | `id` of the element.                                                                  |
```

Comment:
Does it make sense to add `id` here if it's not being used anywhere in SeatLegend?